aep/general/0003/aep.md.j2

```diff
@@ -99,6 +105,22 @@ Examples of complexities that declarative clients abstract away include:
 
 [Terraform][] is an example of such a client.
 
+### Root collection
+
+A [collection](#collection) that does not have any parent collections. For example,
+`/users/`.
+
+### Root Resource
+
+A root resource is a resource which has a collection that has no parents. In
+other words, it is a resource that has a collection that appears at the root of
+a resource hierarchy.
+
+A root resource may also appear as a child of another resource: for example, a
+user permission may exist globally, but also as a child of a individual object.
```

Comment:
FWIW, I don't think this is the clearest example.  I can't really imagine what a global user permission would actually represent.  Not a blocking comment...but FWIW, WDYT about using music playlists instead?  You could have a global playlist like "top trending" or "new this week", but also a playlist under a user (curated by that user) or under a game (the music used in that game).

Reply:
I think the playlist one is fine, so I'll add that.

But for permissions - I think it is often that you have an organization-level permission, along with a folder-level permission in a nested organization, etc. I guess it's true that you would never have a permission without some sort of parent, so I agree your example is better.

Reply:
update, PTAL!